Extension/package.json

```diff
@@ -2327,6 +2327,24 @@
           "markdownDescription": "%c_cpp.configuration.simplifyStructuredComments.markdownDescription%",
           "scope": "application"
         },
+        "C_Cpp.autocompleteDoxygenComment":{
```

Comment:
The formatting style of `package.json` has a space after the `:`. Please add spaces after `:`.

Reply:
You should be able to run VS Code's "Format Document" command, as long as your formatting settings are as expected.